src/raft.rs

```diff
@@ -296,7 +287,7 @@ impl<T: Storage> Raft<T> {
             "{} newRaft [peers: {:?}, term: {:?}, commit: {}, applied: {}, last_index: {}, \
              last_term: {}]",
             r.tag,
-            r.prs().nodes(),
+            r.prs().voters().collect::<Vec<_>>(),
```

Comment:
Is `r.prs().voters()` sufficient enough?

Reply:
Before `.nodes()` only returned voters already.